.changesets/feat_uuid_unix_timestamp_rhai_functions.md

```diff
@@ -0,0 +1,11 @@
+### Support for UUID and Unix timestamp functions in Rhai 
+
+*Description*
+
+When building Rhai scripts, it's often that you'll need to add headers that either uniquely identify a request, or append timestamp information for processing information later, such as crafting a trace header or otherwise. 
+
+While the default `timestamp()` and similar functions (e.g. `apollo_start`) can be used for a somewhat similar function, it also isn't able to be translated into an epoch. As a result, it doesn't acutely address the asks we've heard from users.
+
+This adds a `uuid_v4()` and `unix_now()` function to obtain a UUID and Unix timestamp, respectively.
+
```

Comment:
Just dropping by to say that this was an **excellent** changelog entry, @lleadbet.  Thanks for that!

Reply:
hi there, I am trying to use this function under supergrpah_service but I keep getting ` service callback failed: Function not found: uuid_v4 () (line 4, position 14) in call to function supergraph_service` 
heres my script: 
```rhai
// At the supergraph_service stage, register callbacks for processing requests and
// responses.
fn supergraph_service(service) {
    let id = uuid_v4();
    print(id);
```
Is there any thing needed to import to use this function?